package.json

```diff
@@ -36,6 +36,7 @@
     "react": "16.11.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz",
     "react-native-gesture-handler": "~1.8.0",
+    "react-native-joi": "0.0.5",
```

Comment:
Why do we need this? Cannot use joi like in the backend?

Reply:
the backend `joi` library is not compatible for mobile application.

Reply:
![File](https://user-images.githubusercontent.com/66373739/103716536-47cf7d80-4ffe-11eb-9a69-97d9fa855cf6.jpg)

Reply:
Oic, there is a @hapi/joi in our dependencies, can you check if that is used? Else can remove

Reply:
`@hapi/joi` was moved to standalone `joi` package. which is not compatible for front-end.

Reply:
What I mean is in `package.json` in this repo there is @hapi/joi, if we are not using that, please help to remove it. Thank you!

Reply:
@hapi/joi is not there in package.json. plz check it.

Reply:
Ah my bad, it is in the lock file, not package.json